neurovault/urls.py

```diff
@@ -409,6 +419,21 @@ def images(self, request, pk=None):
             page, context={'request': request}, many=True)
         return paginator.get_paginated_response(serializer.data)
 
+    def add_image(self, request, pk=None):
+        collection = get_collection(pk, request, mode='api')
+
+        if not owner_or_contrib(request, collection):
+            self.permission_denied(request)
+
+        image = StatisticMap(collection=collection)
+        serializer = EditableStatisticMapSerializer(data=request.data,
+                                                    instance=image)
```

Comment:
This works only for StatisticalMaps. What about Atlases and NIDM-Results? Maybe we should have different endpoints for each class.